api-2.2/pom.xml

```diff
@@ -0,0 +1,102 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.openmrs.module</groupId>
+		<artifactId>reporting</artifactId>
+		<version>1.19.0-SNAPSHOT</version>
+	</parent> 
+
+	<artifactId>reporting-api-2.2</artifactId>
+	<packaging>jar</packaging>
+	<name>reporting Module api 2.2</name>
+	<description>2.2 api project for reporting</description>
+	
+	<properties>
+		<openMRSVersion>2.2.0</openMRSVersion>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>${project.parent.groupId}</groupId>
+			<artifactId>${project.parent.artifactId}-api</artifactId>
+			<version>${project.parent.version}</version>
+			<scope>provided</scope>
+		</dependency>	
+		<dependency>
+			<groupId>${project.parent.groupId}</groupId>
+			<artifactId>${project.parent.artifactId}-api-tests</artifactId>
+			<version>${project.parent.version}</version>
+			<type>test-jar</type>
+			<scope>test</scope>
+		</dependency>  
+ 
+          <dependency>
+			<groupId>org.openmrs.test</groupId>
+			<artifactId>openmrs-test</artifactId>
+			<type>pom</type>
+			<scope>test</scope>
+			<version>${openMRSVersion}</version> 
+		  </dependency>  
+		<dependency>
+			<groupId>org.openmrs.api</groupId>
+			<artifactId>openmrs-api</artifactId>
+			<version>${openMRSVersion}</version>
+		</dependency>
+		
+		<dependency>
+			<groupId>org.openmrs.api</groupId>
+			<artifactId>openmrs-api</artifactId>
+			<type>test-jar</type>
+			<scope>test</scope>
+			<version>${openMRSVersion}</version>
+		</dependency>
+
+         <dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>reportingcompatibility-api</artifactId>
+			<version>2.0.7-SNAPSHOT</version>
+			<scope>provided</scope>
```

Comment:
@mozzy11 - can you explain this dependency on the reporting compatibility module?  Is this needed?

Reply:
Yes its needed otherwise , Test fail at runtime with"NoClassDefFoundError" >>AbstractReportObject

Reply:
@mozzy11 .  hmm.  I see this is also a dependency in the api pom.  Although in that pom it refers to version 1.5.8.  We should be consistent and use a variable most likely.  I wonder if it would make sense to make this a test scoped dependency in both places?

Reply:
i used a variable for the version , however , i only made the dependence in api2_2 test scoped